python/tests/kat/t_hosts.py

```diff
@@ -716,7 +716,7 @@ def manifests(self) -> str:
     authority: none
   mappingSelector:
     matchLabels:
-      hostname: tls-context-host-1
+      host-one: tls-context-host-1
```

Comment:
I just renamed this from `hostname` to `host-one` since the third `Host` in this test uses the `hostname` label for it's `matchLabels` selector, and with the fixed logic, the shared `Mapping` between those `Hosts` can't have the same label twice which is now a strict requirement. The renaming just avoids collision on the `hostname` label.